package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gl-matrix",
   "description": "Javascript Matrix and Vector library for High Performance WebGL apps",
-  "version": "2.3.2",
+  "version": "2.4.0",
   "main": "src/gl-matrix.js",
```

Comment:
I like the pattern of the "main" entry being the `dist/` built file -- this lets others import via node or webpack/browserify still, but not need to assume any sort of ES support (and even if they use babel, they'd need to make sure that they have the same babel configs as you, although limited to the es2015 preset, less of an issue).

An importer would still need the same build environment as the project however if someone is only importing per module (`gl-matrix/src/gl-matrix/mat3` vs `gl-matrix`), but that's the cost to them for selectively importing.

Any reason against using `"main": "dist/gl-matrix.js",`?